### DISCUSSION TITLE
Open Intune Baseline policies imported, visible in Intune Management Tool after clicking Load All, but not visible in Intune Portal.

### DISCUSSION THREAD

**Discussion by Nenad-Arch** [Micke-K/IntuneManagement#252] [Q&A] [answered]
Dear community, 
I've successfully imported most of the Open Intune Baseline policies using this tool, but as the tittle says, I can't view all of them in my Intune Portal. In particular, I'm unable to see Defender Antivirus, Defender Updates, Firewall, BitLocker as per below:
<img width="453" alt="image" src="https://github.com/Micke-K/IntuneManagement/assets/175128357/01c702f0-0671-49c8-8617-5a92941f0fbc">
What I see in Intune are all policies except those on the photo above. All of them are multiplied because I imported again thinking they haven't been imported correctly. 
If someone can please help, that would be amazing. Thank you!

**Comment by Micke-K** [comment]
Hello,

Did you look under Endpoint Security?

These policies uses the same API as Settings Catalog policies but they are located in other nodes in the Intune portal. 

Cheers!

**Comment by Nenad-Arch** [comment]
Hey Mikael!

Thank you for quick reply.

Yes, I’ve looked there as well but nothing.

Thank you.

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by Micke-K** [comment, marked as answer]
Hello,

I imported all Settings Catalog policies. The Defender Antivirus policies are available in the Antivirus node in the portal. They should be there if you can see them in the Tool. 

It could also be permissions eg you don't have permissions to view Endpoint Security policies. This could happen if you use Intune Roles and Role Assignments in your environment.

Cheers!

![image](https://github.com/Micke-K/IntuneManagement/assets/43226266/377493e7-aa3d-4737-80b6-d7ddf449a0af)

**Reply by Nenad-Arch** [reply]
That worked, Antivirus, Disk encryption and Firewall! Me being very, very dumb, but thank you so much for your help!

**Comment by Nenad-Arch** [comment]
Hello Mikael,

I’ll check it out a bit later and reply back.

Thank you so much!


Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________
